[Homework #5] - css-advanced by Max Hirning

Max-Hirning opened the pull request:
Added button to top and theme icon changing

ilya-ischenko commented on homeworks/max-hirning/04-css/index.html:
```diff
@@ -0,0 +1,140 @@
+                <div class="profile_info">
+                    <h1 class="title">Victor Alvarado</h1>
+                    <p class="text profile_info_position">Software Engineer</p>
+                    <nav class="social_media">
+                        <a class="social_media_link" target="_blank" href="https://github.com/Max-Hirning">
+                            <img class="social_media_link_icon" src="./assets/icons/github.svg" alt="github">
+                        </a>
+                        <a class="social_media_link" target="_blank" href="https://twitter.com/">
+                            <img class="social_media_link_icon" src="./assets/icons/twitter.svg" alt="twitter">
+                        </a>
+                        <a class="social_media_link" target="_blank" href="https://www.linkedin.com/in/max-hirning-2bbb03230/">
+                            <img class="social_media_link_icon" src="./assets/icons/linkedin.svg" alt="linkedin">
+                        </a>
+                        <a class="social_media_link" target="_blank" href="https://www.youtube.com/">
+                            <img class="social_media_link_icon" src="./assets/icons/youtube.svg" alt="youtube">
+                        </a>
+                    </nav>
```
BEM:
social-media
  social-media__link
    social-media__link-icon

ilya-ischenko reviewed (commented):
1. I see that you are trying to push hw4 instead of hw5
2. In HW5 class names should follow BEM approach - https://en.bem.info/methodology/quick-start/#element. You do not follow this approach in your homework.

Max-Hirning commented:
Fixed issue with BEM part.

About that, I am pushing hw4 and hw5. The thing is that I fixed hw4 by your remarks and was sure that there will anything left to correct, so I started hw5. Finished it before hw4 was pushed to main.

Hw4 is already in main branch and this pr pushes hw4(which wasn't changed) and hw5. In result nothing will break

ilya-ischenko commented:
> Fixed issue with BEM part.
> 
> About that, I am pushing hw4 and hw5. The thing is that I fixed hw4 by your remarks and was sure that there will anything left to correct, so I started hw5. Finished it before hw4 was pushed to main.
> 
> Hw4 is already in main branch and this pr pushes hw4(which wasn't changed) and hw5. In result nothing will break
